src/views/Order.vue

```diff
@@ -240,7 +251,29 @@
                       </ion-buttons> -->
                     </ion-list>
                   </ion-card>
-                  
+                  <ion-card>
+                    <ion-card-header>
+                      <ion-card-title>{{ $t("Fulfillment") }}</ion-card-title>
```

Comment:
why is the entire card being added here? The expected change should just be the addition of the card header right?

Reply:
I think it was pushed by mistake during resolving a merge conflict. I have removed the extra code and updated the PR.